src/services/art_generator.py

```
@@ -0,0 +1,220 @@
+    all they once loved.""",
+    "item": """Artifacts of power in this darkened realm take two forms: those
+    corrupted by the God of Hope's touch, glowing with insidious golden light and
+    whispering false promises; and those forged in pure darkness, their surfaces
+    drinking in light itself. The corrupted items offer tremendous power at the cost
+    of slowly succumbing to hope's curse, while shadow-forged gear helps resist the
+    spreading taint.""",
+}
+
+
+def _generate_art(
+    prompt: str, config: ArtGenerationConfig = ArtGenerationConfig()
```

Ruff (0.8.0)

Do not perform function call `ArtGenerationConfig` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)